# HG changeset patch
# User Rik <rik@octave.org>
# Date 1345005146 25200
#      Tue Aug 14 21:32:26 2012 -0700
# Node ID 5984aa619cd9a86ca9be9d5e2500559a59a1456c
# Parent  f3a5338aaa0122944fe393d25f6f51bf5b3f0abd
doc: Remove multiple seealso links to I (bug #37046).

arith.txi: Remove multiple doc-anchors to I

data.cc: Remove multiple seealso links (i,j,J) to I.

diff --git a/doc/interpreter/arith.txi b/doc/interpreter/arith.txi
--- a/doc/interpreter/arith.txi
+++ b/doc/interpreter/arith.txi
@@ -304,20 +304,16 @@ cos (pi/2)
 
 @node Mathematical Constants
 @section Mathematical Constants
 
 @DOCSTRING(e)
 
 @DOCSTRING(pi)
 
-@c Provide a Seealso link location for these objects in the documentation
-@anchor{doc-i}
-@anchor{doc-j}
-@anchor{doc-J}
 @DOCSTRING(I)
 
 @DOCSTRING(Inf)
 
 @DOCSTRING(NaN)
 
 @DOCSTRING(eps)
 
diff --git a/src/interpfcn/data.cc b/src/interpfcn/data.cc
--- a/src/interpfcn/data.cc
+++ b/src/interpfcn/data.cc
@@ -4055,17 +4055,17 @@ to the base of natural logarithms.  The 
 \n\
 When called with no arguments, return a scalar with the value @math{e}.  When\n\
 called with a single argument, return a square matrix with the dimension\n\
 specified.  When called with more than one scalar argument the first two\n\
 arguments are taken as the number of rows and columns and any further\n\
 arguments specify additional matrix dimensions.\n\
 The optional argument @var{class} specifies the return type and may be\n\
 either \"double\" or \"single\".\n\
-@seealso{log, exp, pi, i, j}\n\
+@seealso{log, exp, pi, I}\n\
 @end deftypefn")
 {
 #if defined (M_E)
   double e_val = M_E;
 #else
   double e_val = exp (1.0);
 #endif
 
@@ -4212,17 +4212,17 @@ When called with no arguments, return a 
 pi.\n\
 @end ifnottex\n\
 When called with a single argument, return a square matrix with the dimension\n\
 specified.  When called with more than one scalar argument the first two\n\
 arguments are taken as the number of rows and columns and any further\n\
 arguments specify additional matrix dimensions.\n\
 The optional argument @var{class} specifies the return type and may be\n\
 either \"double\" or \"single\".\n\
-@seealso{e, i, j}\n\
+@seealso{e, I}\n\
 @end deftypefn")
 {
 #if defined (M_PI)
   double pi_val = M_PI;
 #else
   double pi_val = 4.0 * atan (1.0);
 #endif
 
@@ -4321,17 +4321,17 @@ be reused for other purposes (such as i 
 \n\
 When called with no arguments, return a scalar with the value @math{i}.  When\n\
 called with a single argument, return a square matrix with the dimension\n\
 specified.  When called with more than one scalar argument the first two\n\
 arguments are taken as the number of rows and columns and any further\n\
 arguments specify additional matrix dimensions.\n\
 The optional argument @var{class} specifies the return type and may be\n\
 either \"double\" or \"single\".\n\
-@seealso{e, pi, log, exp, i, j, J}\n\
+@seealso{e, pi, log, exp}\n\
 @end deftypefn")
 {
   return fill_matrix (args, Complex (0.0, 1.0), "I");
 }
 
 DEFALIAS (i, I);
 DEFALIAS (J, I);
 DEFALIAS (j, I);
